L1Trigger/TrackFindingTMTT/src/KFbase.cc

```diff
@@ -811,9 +824,10 @@ namespace tmtt {
       kfEtaReg = iEtaReg - numEtaRegions_ / 2;
     }
 
-    bool ambiguous = ambiguityMap[kfEtaReg][kfLayer];
-
-    //bool ambiguous = false;
+    bool ambiguous = false;
+    if (settings_->KFUseMaybeLayers()) ambiguous = ambiguityMap[kfEtaReg][kfLayer];
+    cout<<ambiguous<<endl;
```

Comment:
Please remove "cout" statement. It is printing hundreds of times per job. We don't print most of the other cfg params, so it is not needed. (And in any case, use of "cout" is forbidden in CMSSW, since it is not threadsafe. One must use MessageLogger instead).